### PR TITLE
refactor(cli): remove old cli commands

### PR DESCRIPTION
**Pull request by laurentiuNiculae**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/cli/cve_cmd_test.go [hunk 1]
```diff
@@ -174,10 +179,10 @@ func TestSearchCVECmd(t *testing.T) {
 	})
 
 	Convey("Test CVE by name and CVE ID - long option", t, func() {
-		args := []string{"cvetest", "--image", "dummyImageName", "--cve-id", "aCVEID", "--url", "someURL"}
+		args := []string{"cveid", "CVE-CVEID", "--repo", "dummyImageName", "--url", baseURL, "--config", "cvetest"}
```
[the remark this answers is not in the document]
Comment (laurentiuNiculae) [reply pkg/cli/cve_cmd_test.go:182]: There isn't a command that goes "zli cve cve", the command for looking up images by a CVEID is "zli cve cveid"

[the remark this answers is not in the document]
Comment (laurentiuNiculae) [reply pkg/cli/cve_cmd_test.go:182]: Also note that we call NewCVECommand() and not NewCliRoot()

File: pkg/cli/cve_cmd_test.go [hunk 2]
```diff
@@ -44,11 +42,31 @@ import (
...
+	conf.Storage.RootDirectory = rootDir
+
+	defaultVal := true
+	conf.Extensions = &extconf.ExtensionConfig{
+		Search: &extconf.SearchConfig{
+			BaseConfig: extconf.BaseConfig{Enable: &defaultVal},
+		},
+	}
+
+	ctlr := api.NewController(conf)
```
[the remark this answers is not in the document]
Comment (laurentiuNiculae) [reply pkg/cli/cve_cmd_test.go:59]: Yes, it's used to test the presence of GQL queries or not.

File: pkg/cli/cve_cmd_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (andaaron) [reply pkg/cli/cve_cmd_test.go:182]: Why not `zli cve vulnerable`? Or something. I though the CVE id is only passed through optional flags such as `--cve-id` https://github.com/project-zot/zot/blob/main/pkg/cli/cmdflags/flags.go#L12
BTW are those flags still used?

File: pkg/cli/image_cmd_test.go [hunk 3]
```diff
@@ -208,59 +207,26 @@ func TestSearchImageCmd(t *testing.T) {
...
-		args := []string{"imagetest", "--digest", "6e2f80bf", "--url", "someUrlImage"}
-		configPath := makeConfigFile(`{"configs":[{"_name":"imagetest","showspinner":false}]}`)
-		defer os.Remove(configPath)
-		imageCmd := NewImageCommand(new(mockService))
-		buff := bytes.NewBufferString("")
-		imageCmd.SetOut(buff)
-		imageCmd.SetErr(buff)
-		imageCmd.SetArgs(args)
-		err := imageCmd.Execute()
+		searchConfig := getTestSearchConfig("http://127.0.0.1:8080", new(mockService))
```
[the remark this answers is not in the document]
Comment (andaaron) [reply pkg/cli/image_cmd_test.go:213]: If it was used, it would have needed a check to see if it is open.

**Review by andaaron — APPROVED** [review]

**Review by rchincha — APPROVED** [review]
lgtm

**Comment by rchincha** [comment]
This needs to be documented 
cc: @mbshields

**Pull request merged**